### PR TITLE
Enable windows and osx tests

### PR DESCRIPTION
**Pull request by jakemac53**
Successor to https://github.com/dart-lang/samples/pull/47

Fixes https://github.com/dart-lang/samples/issues/46

Note that the FFI tests do not run on windows/osx, so those are excluded.

cc @irvifa

### CODE REVIEW

File: mono_repo.yaml [hunk 1]
```diff
@@ -3,3 +3,7 @@ travis:
     apt:
       packages:
         - cmake
+
+# Merges all analyze stages into one bot, which is more efficient
+merge_stages:
+  - analyze
```
Comment (kevmoo) [inline mono_repo.yaml:9]: newline!

**Comment by jakemac53** [comment]
ping @domesticmouse

**Comment by irvifa** [comment]
Thank you @jakemac53 now I learn how to use `mono_repo` 🙂

**Comment by domesticmouse** [comment]
@jakemac53 I believe you have squash & merge access to this repo, if not ping me to land this PR.

**Pull request merged**